pieforproviders/pieforproviders
# Pie 1717/multiple check ins

## 💅🏼 What issue does this fix?



4th checkbox on #1717 

## 🍂 Before You Submit


## 🛷 Deployment


## 🧵 Steps to set up locally


## 🧳 Steps to test
- `bundle exec rails db:migrate` - all attendances should have a newly-created service day associated to them
- create an attendance; a service day should be created automatically
- create another attendance for the same day as the last one; the service day count should not change and the new attendance should be associated to the same service day as the last one

## 🕯 Caveats, concerns
